src/components/policy/policy_external/src/cache_manager.cc

```diff
@@ -2300,13 +2301,29 @@ bool CacheManager::SetMetaInfo(const std::string& ccpu_version,
   return true;
 }
 
+bool CacheManager::SetHardwareVersion(const std::string& hardware_version) {
+  CACHE_MANAGER_CHECK(false);
+  sync_primitives::AutoLock auto_lock(cache_lock_);
+
+  *pt_->policy_table.module_meta->hardware_version = hardware_version;
+  Backup();
+  return true;
```

Comment:
Always true? Please make this function void.

Reply:
It may be false because of `CACHE_MANAGER_CHECK(false)`

Reply:
Oh, I see. But return value is'nt checked anywhere.